### PR TITLE
[DOCS] IP

### PR DESCRIPTION
**Pull request by mhilbrunner**

### CODE REVIEW

File: doc/classes/IP.xml [hunk 1]
```diff
@@ -79,24 +79,34 @@
...
+			DNS hostname resolver status: Waiting.
 		</constant>
 		<constant name="RESOLVER_STATUS_DONE" value="2" enum="ResolverStatus">
+			DNS hostname resolver status: Done.
 		</constant>
 		<constant name="RESOLVER_STATUS_ERROR" value="3" enum="ResolverStatus">
+			DNS hostname resolver status: Error.
 		</constant>
 		<constant name="RESOLVER_MAX_QUERIES" value="32">
+			Maximum number of concurrent DNS resolver queries allowed, [code]RESOLVER_INVALID_ID is returned if exceeded[/code].
```
Comment (cbscribe) [inline doc/classes/IP.xml]: Misplaced [/code] tag.

**Comment by mhilbrunner** [comment]
Good catch, fixed

File: doc/classes/IP.xml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="IP" inherits="Object" category="Core" version="3.0-beta">
 	<brief_description>
-		IP Protocol support functions.
+		IP Protocol support functions like DNS resolution.
 	</brief_description>
 	<description>
-		IP contains support functions for the IPv4 protocol. TCP/IP support is in different classes (see [StreamPeerTCP] and [TCP_Server]). IP provides hostname resolution support, both blocking and threaded.
+		IP contains support functions for the IPv4 protocol. TCP/IP support is in different classes (see [StreamPeerTCP] and [TCP_Server]). IP provides DNS hostname resolution support, both blocking and threaded.
```
Comment (cbscribe) [inline doc/classes/IP.xml]: What about TYPE_IPV6?

Comment (mhilbrunner) [reply doc/classes/IP.xml]: I'll drop the version type, as IPv6 support was added in the meanwhile, and together they are the only relevant versions in >99% of use cases.

**Comment by mhilbrunner** [comment]
Fixed

File: doc/classes/IP.xml [hunk 3]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="IP" inherits="Object" category="Core" version="3.0-beta">
 	<brief_description>
-		IP Protocol support functions.
+		IP Protocol support functions like DNS resolution.
 	</brief_description>
 	<description>
-		IP contains support functions for the IPv4 protocol. TCP/IP support is in different classes (see [StreamPeerTCP] and [TCP_Server]). IP provides hostname resolution support, both blocking and threaded.
+		IP contains support functions for the Internet Protocl (IP). TCP/IP support is in different classes (see [StreamPeerTCP] and [TCP_Server]). IP provides DNS hostname resolution support, both blocking and threaded.
```
Comment (karroffel) [inline doc/classes/IP.xml]: Internet **Protocol** 😛

Comment (mhilbrunner) [reply doc/classes/IP.xml]: Damn, why didn't I see that. Thanks. :)

**Pull request merged**